packages/engine/src/worker/runner/python/sender.rs

```diff
@@ -118,6 +118,8 @@ impl NngSender {
 
 impl Drop for NngSender {
     fn drop(&mut self) {
+        // Python has to close the socket on its side for the "-topy*" file to be deleted.
+        let _ = self.send(None, &InboundToRunnerMsgPayload::TerminateRunner, &None);
```

Comment:
The only reason to fail listed is: "The Aio already has a running operation." which we prevent.
I guess it would also fail if the socket is closed, not sure.

It already prints a warning.

Reply:
What warning is printed here? We ignoring the returned result from `NngSender::send`.

Reply:
That doesn't return any error on `InboundToRunnerMsgPayload::TerminateRunner`.